frontend/test/__support__/settings.ts

```diff
@@ -0,0 +1,13 @@
+import MetabaseSettings from "metabase/lib/settings";
+import { createMockSettings } from "metabase-types/api/mocks";
+import { createMockSettingsState } from "metabase-types/store/mocks";
+import type { Settings } from "metabase-types/api";
+
+export function mockSettings(params: Partial<Settings> = {}) {
+  const settings = createMockSettings(params);
+  const state = createMockSettingsState({ values: settings });
+
+  MetabaseSettings.setAll(settings);
```

Comment:
@kulyk should we use `MetabaseSettings.setAll` directly?

Reply:
I think so? `MetabaseSettings` seems to keep a settings values object inside of itself, i.e. it's no synced with the redux state or whatever. AFAIK, normally we call `MetabaseSettings.setAll` anytime we refetch the settings to keep stuff in sync. Looks like a similar use-case. Please LMK if I'm missing something here

Reply:
@kulyk sounds good to me here.